doc/source/appendixb.rst

```diff
@@ -1233,7 +1233,7 @@ large extended sources (larger than about 80 pixels in size)
 because the extended IR scattering wings in these bands,
 which do not affect the photometry of point sources, begin to be
 included. The modified curves are shown in an
-`updated SDSS system response plot <http://www.sdss.org/dr3/instruments/imager/#filters>`_.
+`updated SDSS system response plot <http://classic.sdss.org/dr7/instruments/imager/#filters>`_.
```

Comment:
This is not related to Zope. And I don't really know if updating this link reflects what is actually in CDBS. You have to confirm with RedCat first. Please revert `dr7` change from this PR.

Reply:
Reverted `dr7` in force-push below.